src/main/resources/templates/Landing.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+  <title>Online HTML Editor</title>
+  <link rel="stylesheet" href="mystyle.css">
+
+  <head>
+    <div class="textBox">
+      <h1 class="primaryTextCenter">Welcome to Mountain Man's Millstone</h1>
+    </div>

```

Comment:
This div should be within the <body> tag. The <head> tag is meant for only metadata, and link-like items, and the title.